README.md

```diff
@@ -16,7 +16,7 @@ You tap on left side of screen to turn left.
 
 ### Dual
 
-This control uses left tap to move snake in West and South direction and right tap to move snake to North and East  direction.
+This control uses left tap to move snake in West and South direction i.e.(left and down) and right tap to move snake to North and East  direction i.e.(right and up).
```

Comment:
Shouldn't it be **i.e.(up and right)** so that it remains respective to North and East?

Reply:
Yes , it should be (up and right).